docs/rofl.md

```diff
@@ -240,6 +264,35 @@ offer. Specify the extension period with [`--term`][term-flags] and
 
 [term-flags]: #deploy
 
+### Show machine logs {#machine-logs}
+
+You can fetch logs from your running ROFL app using `oasis rofl machine logs`.
+
+```shell
+oasis rofl machine logs
+```
+
+:::danger Logs are not encrypted!
+
+While only the app admin can access the logs, they are stored
+**unencrypted on the ROFL node**. In production, make sure
+you never print any confidential data to the standard or error outputs!
+
+:::
+
+### Restart a machine {#machine-restart}
+
+To restart a running machine, use `oasis rofl machine restart`.
+
```

Comment:
There is the `--wipe-storage` flag we forgot to describe under the `rofl deploy` command above around line 223.

```
- `--wipe-storage` will wipe any existing persistent storage on the machine
```

This flag is also available here, so I'd put:

```suggestion

If you wish to clear the machine's persistent storage, pass the [`--wipe-storage`] flag.

[`--wipe-storage`]: #machine-deploy
```